templates/travis-ci/.travis.yml.tmpl

```diff
@@ -18,27 +39,55 @@ before_script:
 {{ range $profileName, $profile:= .AWSProfiles}}
   - aws configure set profile.{{ $profileName }}.role_arn arn:aws:iam::{{ $profile.AccountID }}:role/{{ $profile.RoleName}}
   - aws configure set profile.{{ $profileName }}.source_profile _idacct
-  - aws --profile {{ $profileName }} sts get-caller-identity
+  - {{- if $travisCI.Buildevents }} buildevents cmd $BUILDEVENTS_BUILD_ID $STEP_SPAN_ID sts-caller-identity.{{ $profileName }} -- {{- end }} aws --profile {{ $profileName }} sts get-caller-identity
 {{ end }}
+{{- if .Buildevents }}
+  - buildevents step $BUILDEVENTS_BUILD_ID $STEP_SPAN_ID $STEP_START $STEP_NAME
+{{- end }}
 
 jobs:
   include:
     - stage: fogg-apply clean
       script:
         - |
           set -e
+          {{- if .Buildevents }}
+          STEP_START=$(date +%s)
+          STEP_NAME=fogg-apply-clean.${TRAVIS_JOB_NUMBER}
+          STEP_SPAN_ID=$(echo ${STEP_NAME} | sum | cut -f 1 -d \ )
+          {{- end }}
           .fogg/bin/fogg apply
           if [ "$(git status --porcelain)" != "" ]; then
             echo "git tree is dirty after running fogg apply"
             echo "ensure you run fogg apply on your branch"
             git status
             exit 1
           fi
-{{ range $testBucket := .TestBuckets }}
+          {{- if .Buildevents }}
+          buildevents step $BUILDEVENTS_BUILD_ID $STEP_SPAN_ID $STEP_START $STEP_NAME
+          {{- end }}
+{{- $travisCI := . }}
```

Comment:
Is there a reason this is here and at the beginning of the file?

Reply:
Good catch